### ISSUE TITLE
Unable to resolve dependencies from the maven repository located on aws s3

### ISSUE THREAD

**Issue by vorobyov-denis** [jmfayard/refreshVersions#279]
We have a maven repository with credentials located on aws s3 where we store libraries. refreshVersions does not find artifacts stored in such repositories. Are there any plans to support such repositories?

**Comment by LouisCAD** [comment]
Yes, we would like to support everything Gradle and Kotlin support.

It's not in our priority list yet, but we are welcoming any help if someone wants to have it happen sooner. We can guide to where to look at in the codebase for integration.

**Comment by dukesta3** [comment]
I wouldn't mind taking this on as I have a maven repo at work that I have to hardcore versions for.  I know nothing about gradle development though.  If someone can point me to how to get started on this, I'll give it a shot.

**Comment by dukesta3** [comment]
@LouisCAD - I've never done any gradle dev work, so gonna go through https://github.com/jmfayard/refreshVersions/blob/master/CONTRIBUTING.md and make sure I can develop and test locally on my machine.  If you could point where to look at in the codebase for integration, that would be awesome.

**Comment by LouisCAD** [comment]
@dukesta3 This contributing guide is outdated, and we're not done making the new one unfortunately 😞

**Comment by LouisCAD** [comment]
I'n quite busy doing #281 and other things on the project besides day job and life, but I'll make sure to let you know when the guide is updated, and I'll pinpoint where to look at in the codebase.
